src/inverse-kinematics/include/private/InverseKinematicsData.h

```diff
@@ -105,7 +105,15 @@ class internal::kinematics::InverseKinematicsData {
 
     TransformMap m_constraints; /*!< list of hard constraints */
     TransformMap m_targets; /*!< list of targets */
+
+    // Attributes relative to the COM Projection constraint (TODO(traversaro): move most of them in a constraint-specific class)
     iDynTree::ConvexHullProjectionConstraint m_comHullConstraint; /*!< Helper to implement COM constraint */
+    iDynTree::Vector3 m_comHullConstraint_projDirection;
```

Comment:
I am not a big fan of the underscore

Reply:
It is ugly on purpose: those attributes need to be moved to a proper constraint class, and to identify them I added the `m_comHullConstraint_` prefix.

Reply:
See the `TODO` comment on top.